### PR TITLE
Create Vignette structure and first draft

### PR DESCRIPTION
**Pull request by ranjitprakash1986**
Created structure and first draft, some description may need enhancements if we do code improvements.

### CODE REVIEW

**Comment by ranjitprakash1986** [comment]
Hi I have made pushed this branch, please note there may be enhancements needed on the writing based on any code improvements we plan. 
We can start work on the pkgdown now.
NOTE: I had a merge conflict on the "Description" file. covr was added by Eric, my vignette added Knittr.
I have resolved it an requested merge, please check this and if any doubt lets have a call

File: vignettes/sentimentanalyzerR.Rmd [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+```{r load_library}
+library(sentimentanalyzerR)
+```
+
+## Definitions
+
+Compound Score - A score in the range of -1 to 1 that represents the general opinion of a document. It is computed using the valence score for each word in the document. Valence score is a score calculated for each word based on the connotation and context of its usage in a sentence.
+
+Likert Scale - Likert scale is used in questionnaires to enable the respondents to provide a level of agreement or disagreement on a symmetric scale. The scale can be orientated either in the increasing or decreasing order. It has found wide applications in the field of statistics, psychology, business and sciences. Each level of the scale is perceived as evenly spaced in the intensity of agreement/disagreement.
+
```
Comment (erictsai1208) [inline vignettes/sentimentanalyzerR.Rmd:32]: Do we also need to add some detail about `sentiment_score_plot` here?

**Comment by erictsai1208** [comment]
The PR is causing CI to fail. From looking at the error log, it looks like we're missing a package called `readr`. @ranjitprakash1986 Could you also run `use_package("readr")` in the R Studio terminal and push the changes?

**Comment by ranjitprakash1986** [comment]
Ah ok I see, I used that Library for the vignettes. I will add readr as use package tonight

File: vignettes/sentimentanalyzerR.Rmd (hunk 1 above)
Comment (ranjitprakash1986) [reply vignettes/sentimentanalyzerR.Rmd:32]: I was defining only the terms that are somewhat technical. I describe the functions in their individual sections below. I was wondering, sentiment score plot isn't a term by itself to define. I can include it if it still feels necessary when I get to it tonight, pls let me know

File: vignettes/sentimentanalyzerR.Rmd [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+### sentiment_score_plot
+
+While understanding the overall sentiment is essential, a more insightful representation can be obtained through a histogram depicting the most frequent sentiments seen in the corpus. Each count represents a unique document in the corpus. Thus the plot can communicate how many reviews fall in each range of sentiment scores.
+
+```{r hist-plot, eval = TRUE, results='hide', mesage = FALSE, fig.keep='all'}
+hist_plot <- sentiment_score_plot(df, "text")
+
+```
+
+**TBD** - Scope available to alter code to show the the statement with insights.
```
Comment (erictsai1208) [inline vignettes/sentimentanalyzerR.Rmd:91]: Does this need to be removed?

Comment (ranjitprakash1986) [reply vignettes/sentimentanalyzerR.Rmd:91]: It's like a placeholder. There is a paste statement in the code that prints a summary of the insights, that is not returned at present. Depending on the code changes done I need to update this prior to milestone 4. Is it ok to have placeholder like that

**Review by erictsai1208 — APPROVED** [review]
Looks good.

**Pull request merged**